docs/Contributing/guides/api/README.md

```diff
@@ -0,0 +1,14 @@
+# Fleet API Development Guides
+
+This directory contains guides for API development in Fleet.
+
+## API Development
+
+- [Adding New Endpoints](adding-new-endpoints.md) - Guide for adding new endpoints to the Fleet API
+
+## Related Resources
+
+- [API Versioning](../../reference/api-versioning.md) - Documentation on API versioning
+- [API for Contributors](../../workflows/api-for-contributors.md) - Workflow for API development
```

Comment:
Seems like api-for-contributors shouldn't be in workflows? Should probably be in this directory? Also, the description here isn't accurate, and we could likely get away with a single list here rather than splitting between two lists that serve the same-ish purpose.

Reply:
Thanks this was a bad link. It's actually in `/reference`.